src/figuresClasses.ts

```diff
@@ -2,10 +1,75 @@
+export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+    public shape: string = 'triangle',
```

Comment:
The `shape` property should be of type `'triangle'` to match the `Figure` interface. Consider using a type assertion or changing the type to `'triangle'`.